types/amplifier/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": false,
```

Comment:
Could you please enable back the strictness? Or is there any particular reason why to disabled it?

Reply:
It is disabled in @types/amplify too so I don't see the difference.

I tried enabling it but a lot of tests are failing.